README.md

```diff
@@ -38,12 +39,10 @@ You also need ~10 GB on your hard drive and some free time.
 
 ### Configuring and Building
 
-1. Clone the repo, including submodules
+1. Clone the repo
```

Comment:
I think we don't need to make submodules thing implicit. It can be source for strange things when someone pulls updated submodule from upstream, but doesn't update submodules manually (current cmake script doesn't do that, as I can see)

Reply:
I agree it's a bad idea. Also, PR's main meaning is not about git submodules automation. Reverting.